region/norfair/east/Cathedral Entrance.json

```diff
@@ -306,6 +259,59 @@
         "As a backup, the same Sova can be frozen on the left wall, dropped from the shot blocks again, or frozen on the right wall."
       ]
     },
+    {
+      "link": [3, 1],
+      "name": "Leave Shinecharged",
+      "notable": false,
+      "requires": [
+        "HiJump",
+        "canShinechargeMovementComplex",
+        {"canShineCharge": {
+          "usedTiles": 33,
+          "openEnd": 2
+        }},
+        {"or": [
+          "canPreciseWalljump",
+          "canTrickyJump",
+          {"and": [
+            "SpaceJump",
+            "canDodgeWhileShooting"
```

Comment:
Normally it would mean shooting after spacejumping to make it above the shot block as it breaks, but that is not the case here. So dunno.  I'll take it out.
These leaveShinecharged frames are very weird too.